### ISSUE TITLE
Build fails on OSX

### ISSUE THREAD

**Issue by kipelovets** [codeaholics/node-rdb-tools#2]
Build fails on OSX probably because of case-insensitive HFS (Crc64.o and crc64.o are seen as one file and overwrite one another). Build succeeds on a case-sensitive filesystem on the same computer

```
➜ npm build .

> rdb-tools@0.1.0 install /Users/georgy/Documents/Dev/vendor/node-rdb-tools
> node-gyp configure build

  CXX(target) Release/obj.target/Crc64/src/Crc64.o
  SOLINK_MODULE(target) Release/Crc64.node
duplicate symbol __ZN5Crc644InitEN2v86HandleINS0_6ObjectEEE in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc643NewERKN2v89ArgumentsE in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc644PushERKN2v89ArgumentsE in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc648GetValueERKN2v89ArgumentsE in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZTV5Crc64 in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc64C1Ev in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc64C2Ev in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc64D0Ev in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc64D1Ev in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol __ZN5Crc64D2Ev in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol _init in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
duplicate symbol _Crc64_module in:
    Release/obj.target/Crc64/src/crc64.o
    Release/obj.target/Crc64/src/Crc64.o
ld: 12 duplicate symbols for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [Release/Crc64.node] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/Cellar/node/0.8.17/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:255:23)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:99:17)
gyp ERR! stack     at Process._handle.onexit (child_process.js:678:10)
gyp ERR! System Darwin 13.0.0
gyp ERR! command "node" "/usr/local/Cellar/node/0.8.17/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build"
gyp ERR! cwd /Users/georgy/Documents/Dev/vendor/node-rdb-tools
gyp ERR! node -v v0.8.17
gyp ERR! node-gyp -v v0.8.2
gyp ERR! not ok 
npm ERR! rdb-tools@0.1.0 install: `node-gyp configure build`
npm ERR! `sh "-c" "node-gyp configure build"` failed with 1
npm ERR! 
npm ERR! Failed at the rdb-tools@0.1.0 install script.
npm ERR! This is most likely a problem with the rdb-tools package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp configure build
npm ERR! You can get their info via:
npm ERR!     npm owner ls rdb-tools
npm ERR! There is likely additional logging output above.

npm ERR! System Darwin 13.0.0
npm ERR! command "/usr/local/Cellar/node/0.8.17/bin/node" "/usr/local/bin/npm" "build" "."
npm ERR! cwd /Users/georgy/Documents/Dev/vendor/node-rdb-tools
npm ERR! node -v v0.8.17
npm ERR! npm -v 1.2.0
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /Users/georgy/Documents/Dev/vendor/node-rdb-tools/npm-debug.log
npm ERR! not ok code 0
```

**Comment by codeaholics** [comment]
I'll resist the urge to say "stupid OSX" :-) Since I don't have OSX available to test a fix on, is there any chance you could send a pull request?

**Comment by hannes-ucsc** [comment]
Just rename crc32.c to _crc32.c and change the reference in binding.gyp. Does that really require a pull request?

**Comment by codeaholics** [comment]
It does if you want any kind of guarantee of it actually working instead of me just randomly renaming files without testing that it solves the problem.

**Comment by codeaholics** [comment]
Fixed by #3

**Comment by codeaholics** [comment]
v0.1.1 published to npm with this fix. Sometimes it can take a few hours for things to appear on npm.
